var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -19,6 +19,8 @@ class Mumps(Package):
     variant('float', default=True, description='Activate the compilation of smumps')
     variant('complex', default=True, description='Activate the compilation of cmumps and/or zmumps')
     variant('idx64', default=False, description='Use int64_t/integer*8 as default index type')
+    variant('tests', default=False, description='Run small MPI unit tests')
```

Comment:
one may consider doing it unconditionally when it's non-mpi installation, if that's what you prefer. Then I can do

 ```
variant(`mpi-tests',...)
```
and change the logic at the end to make it run if 
```
`~mpi` or `+mpi-tests`.
```